nautobot/dcim/views.py

```diff
@@ -5172,7 +5172,6 @@ def form_save(self, form, **kwargs):
         url_path="add-member",
         url_name="add_member",
         custom_view_base_action="change",
-        custom_view_additional_permissions=["dcim.change_virtualchassis"],
```

Comment:
Removed as redundant with the default permission already imposed by the viewset + base_action.